src/FrameBuffer.cpp

```diff
@@ -1263,6 +1263,9 @@ bool FrameBufferToRDRAM::_prepareCopy(u32 _startAddress)
 template <typename TSrc, typename TDst>
 void _writeToRdram(TSrc* _src, TDst* _dst, TDst(*converter)(TSrc _c), TSrc _testValue, u32 _xor, u32 _width, u32 _height, u32 _numPixels, u32 _startAddress, u32 _bufferAddress, u32 _bufferSize)
 {
+	const u32 np = sizeof(u32) / sizeof(TDst);
+	TDst buf[sizeof(u32) / sizeof(TDst)];
+	u32* dst32 = reinterpret_cast<u32*>(_dst);
```

Comment:
If it helps, it's not by much. Biggest different is doing the memcpy of the buffer before passing it to _writeToRdram.